tests/test_flows.sh

```diff
@@ -181,8 +181,9 @@ test_channel_selection() {
     
     dbg "Confirming 1905.1 Ack Message request was received on agent"
     # TODO: When creating handler for the ACK message on the agent, replace lookup of this string
-    check_log ${REPEATER1} agent_wlan0 "ACK_MESSAGE"
-    check_log ${REPEATER1} agent_wlan2 "ACK_MESSAGE"
+    # TODO: currently controller sends empty channel selection request, so no switch is performed
+    # check_log ${REPEATER1} agent_wlan0 "ACK_MESSAGE"
+    # check_log ${REPEATER1} agent_wlan2 "ACK_MESSAGE"
```

Comment:
Actually, this was incorrect anyway. Indeed, the agent should send a Channel Selection Response so that's what we have to test on the controller.

Reply:
So can we remove the TODO?